### DISCUSSION TITLE
Better Build Ignore

### DISCUSSION THREAD

**Discussion by ZetiMente** [vercel/vercel#8515] [Ideas] [answered]
![Screen Shot 2022-09-04 at 9 47 22 PM](https://user-images.githubusercontent.com/76985631/188351204-323e9039-39cf-4ad9-a8a8-1cd51533bc9a.png)

Netlify, your competition, has a simple UI for controlling the branches for deployments and preview. It would be great if Vercel could add a similar functionality as an option in the UI. Instead of forcing me to write a shell script. I want the preview deploy only to work on master & develop branch, not on every push to the repo. Vercel assumes I use their product in one particular way, and I appreciate the functionality for env variables. 

![Screen Shot 2022-09-04 at 10 06 49 PM](https://user-images.githubusercontent.com/76985631/188353150-9d4cbe9c-b7cf-484b-87f3-4e51a78bf82b.png)

But I want to have two projects in Vercel, one project is development/test and another as production. Each project aligned with a branch. I tried to make a script to do this based on the documentation but it didn't do what I want. I want to still have my PR from a feature branch to develop do the preview deployment. But since Vercel is looking at a PR as still commiting to the feature branch, this code doesn't work. 

```
#!/bin/bash
if [[ "$VERCEL_GIT_COMMIT_REF" == "master" || "$VERCEL_GIT_COMMIT_REF" == "develop"  ]] ; then
  exit 0;
else
  exit 1;
fi
```
Thank you Vercel for it's strong support of SvelteKit.

**Comment by amyegan** [comment, marked as answer]
Thank you for sharing this @ZetiMente! I'm adding it to the requested features list for the team to review
